download-dependencies.sh

```diff
@@ -2,7 +2,9 @@
 
 echo "Creating directory"
 mkdir -p ./dependencies && \
-cd ./dependencies && \
+cd ./dependencies
+# Removing prior dependencies
+rm -rf * 
 echo "Downloading dependencies"
 curl -sS https://preview.documentation.bedrock.aws.dev/Documentation/SDK/bedrock-python-sdk.zip > sdk.zip && \
```

Comment:
Have just seen this DNS name is down and heard it might not be back... Will see if I have access to modify & merge this PR directly to get it fixed ASAP.